beangulp/__init__.py

```diff
@@ -257,7 +258,7 @@ def _importer(importer):
 
 
 class Ingest:
-    def __init__(self, importers, hooks=None):
+    def __init__(self, importers: Sequence[Union[Importer, ImporterProtocol]], hooks: Optional[Sequence[Any]] = None) -> None:
```

Comment:
`hooks: Optional[Sequence[Any]]` could at least be `hooks: Optional[Sequence[Callable]]`